src/test/java/api/requests/RequestsAPIRetrievalTests.java

```diff
@@ -361,6 +371,14 @@ public void canGetMultipleRequests() {
 
     requestHasCallNumberStringProperties(requests.getById(
       requestForTemeraire.getId()), "tem");
+
+    assertThat(requests, hasItems(
+      hasJsonPath("patronComments", "Comment 1"),
+      hasJsonPath("patronComments", "Comment 2"),
+      hasJsonPath("patronComments", "Comment 3"),
+      hasJsonPath("patronComments", "Comment 4"),
+      hasJsonPath("patronComments", "Comment 5")
+    ));
```

Comment:
Would this be easier to understand if it was extracted to a method something like `assertThatRequestsHavePatronComments`?

Might it be worth having one of the requests having no comments?

Reply:
Sure, make sense for me.